hub/core/dataset/dataset.py

```diff
@@ -1,5 +1,5 @@
 # type: ignore
-from unittest import skip
+from pandas import DataFrame
```

Comment:
Lets not have a Pandas dependency just for __str__

Reply:
Okay, Let me try implementing the to_string method then. Hope it works.